src/android/Player.java

```diff
@@ -92,6 +92,10 @@ public void onDismiss(DialogInterface dialog) {
         public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event) {
             JSONObject payload = Payload.keyEvent(event);
             new CallbackResponse(Player.this.callbackContext).send(PluginResult.Status.OK, payload, true);
+            
+            //Added method to handle remote key events to show controls on any key event
+            exoView.showController();
```

Comment:
For the moment we are not using the info header, I was trying to implement the same method used in onTouchListener with no results. I'll come back to it when we need to use info header or when I find some time.